app/dashboard/templates/shared/newsletter.html

```diff
@@ -16,7 +16,7 @@
 {% endcomment %}
 {% load i18n static %}
 <link rel="stylesheet" href="{% static "v2/css/newsletter.css" %}">
-<div class="row newsletter text-center" >
+<div class="row newsletter text-center g-font-muli">
```

Comment:
I don't think this will be necessary since whichever page this will be included, the body will already have the font-family styling.